deploy.php

```diff
@@ -0,0 +1,228 @@
+        $remotePort = Configuration::fromRemote()->get('FORK_DATABASE_PORT');
+        $remoteName = Configuration::fromRemote()->get('FORK_DATABASE_NAME');
+        $remoteUser = Configuration::fromRemote()->get('FORK_DATABASE_USER');
+        $remotePassword = Configuration::fromRemote()->get('FORK_DATABASE_PASSWORD');
+
+        $localHost = Configuration::fromLocal()->get('FORK_DATABASE_HOST');
+        $localPort = Configuration::fromLocal()->get('FORK_DATABASE_PORT');
+        $localName = Configuration::fromLocal()->get('FORK_DATABASE_NAME');
+        $localUser = Configuration::fromLocal()->get('FORK_DATABASE_USER');
+        $localPassword = Configuration::fromLocal()->get('FORK_DATABASE_PASSWORD');
+
+        $remoteDatabaseUrl = parse_url("mysql://{$remoteUser}:{$remotePassword}@{$remoteHost}:{$remotePort}/{$remoteName}?serverVersion=5.7&charset=utf8mb4");
+        $localDatabaseUrl = parse_url("mysql://{$localUser}:{$localPassword}@{$localHost}:{$localPort}/{$localName}?serverVersion=5.7&charset=utf8mb4");
+
+        // create a backup
+        // @todo make separate backup dir
+        run(
+            sprintf(
+                'mysqldump --lock-tables=false --set-charset %1$s %2$s > {{deploy_path}}/backup_%3$s.sql',
+                $databaseUtility->getConnectionOptions($remoteDatabaseUrl),
+                $databaseUtility->getNameFromConnectionOptions($remoteDatabaseUrl),
+                date('YmdHi')
+            )
+        );
+
+        runLocally(
+            sprintf(
+                'mysqldump --column-statistics=0 --lock-tables=false --set-charset %1$s %2$s > ./db_upload.tmp.sql',
+                $databaseUtility->getConnectionOptions($localDatabaseUrl),
+                $databaseUtility->getNameFromConnectionOptions($localDatabaseUrl)
+            )
+        );
+
+        upload('./db_upload.tmp.sql', '{{deploy_path}}/db_upload.tmp.sql');
+        runLocally('rm ./db_upload.tmp.sql');
+
+        run(
+            sprintf(
+                'mysql %1$s %2$s < {{deploy_path}}/db_upload.tmp.sql',
+                $databaseUtility->getConnectionOptions($remoteDatabaseUrl),
+                $databaseUtility->getNameFromConnectionOptions($remoteDatabaseUrl)
+            )
+        );
+        run('rm {{deploy_path}}/db_upload.tmp.sql');
+    }
+);
+
+desc('Cleanup the codebase');
+task('sumo:files:cleanup', function () {
+    run('rm -rf {{release_path}}/.github');
+    run('rm -rf {{release_path}}/.git');
+    run('rm -rf {{release_path}}/.gitattributes');
+    run('rm -rf {{release_path}}/Dockerfile');
+    run('rm -rf {{release_path}}/docker-compose.yml');
+    run('rm -rf {{release_path}}/.scrutinizer.yml');
+    run('rm -rf {{release_path}}/.codecov.yml');
+    run('rm -rf {{release_path}}/php.ini');
+    run('rm -rf {{release_path}}/phpunit.xml.dist');
+    run('rm -rf {{release_path}}/phpstan.neon');
+    run('rm -rf {{release_path}}/UPGRADE***');
+    run('rm -rf {{release_path}}/var/docks');
```

Comment:
Wat zijn docks?

Reply:
@jonasdekeukelaere fixed in https://github.com/sumocoders/deployer-sumo-forkcms/blob/master/recipe/cleanup.php